test/run_pass/test_net_10.js

```diff
@@ -30,7 +31,8 @@ socket1.on('timeout', function() {
 });
 
 socket1.on('error', function() {
-  assert.fail();
+  error = true;
```

Comment:
Can we identify what kind of error comes in nuttx as the argument of the event handler? If we could use it, the following could be for thought.

```
error = false;

socket1.on('error',  function(e) {
      if(e.code == 'ErrorOccuredInNuttX') 
          error = true;
   }
}

process.on('exit', function() {
   assert(error || timedout);
   ..
}
```